endaq/calc/shock.py

```diff
@@ -17,29 +17,186 @@
 from endaq.calc import utils
 
 
-def _rel_displ_transfer_func(
-    omega: float, damp: float = 0.0, dt: float = 1.0
-) -> scipy.signal.ltisys.TransferFunctionDiscrete:
+def _abs_accel_coeffs(omega, Q, T):
```

Comment:
abbreviated form: as-is 👍
non-abbreviated form:
```suggestion
def _absolute_acceleration_coeffs(omega, Q, T):
```